src/main/java/org/broadinstitute/consent/http/mail/message/NewProgressReportCaseMessage.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.consent.http.mail.message;
+
+import java.util.Map;
+import org.broadinstitute.consent.http.enumeration.EmailType;
+import org.broadinstitute.consent.http.models.User;
+
+public class NewProgressReportCaseMessage extends MailMessage {
+
+  private static final String NEWCASE_DUL = "Log vote on Data Use Limitations case id: %s.";
+  private static final String NEWCASE_DAR = "Log votes on Data Access Request case id: %s.";
+  private final String type;
+  private final String referenceId;
+
+  public NewProgressReportCaseMessage(User toUser, String referenceId, String type) {
+    super(toUser, EmailType.NEW_PROGRESS_REPORT_CASE);
+    this.referenceId = referenceId;
+    this.type = type;
+  }
+
+  @Override
+  public String createSubject() {
+    if (type.equals("Data Use Limitations")) {
+      return String.format(NEWCASE_DUL, referenceId);
+    } else {
```

Comment:
This should be a defunct case. We don't make DUL elections any more which is where I believe this string comes from. I would prefer that this message handle just the `Data Access Request` case